python/lsst/jointcal/jointcal.py

```diff
@@ -77,6 +90,7 @@ class JointcalConfig(pexConfig.Config):
         doc = "How to select sources for cross-matching",
         default = "astrometry"
     )
+    # TODO: CmdLineTask has a profile thing built-in: can we tie in to that?
```

Comment:
I've dealt with this by adding a "--profile_jointcal" cmdline argument which can be the used with the context manager. I want to be able to profile individual parts of jointcal separately, since I know most of the effort is going into the actual math but there are likely other things that could be sped up independently.